diff --git a/CMakeLists.txt b/CMakeLists.txt
index b754f1e..dcb58a2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,6 +16,10 @@ add_library(vosk
 find_package(kaldi REQUIRED)
 target_link_libraries(vosk PUBLIC kaldi-base kaldi-online2 kaldi-rnnlm fstngram)
 
+if(MSVC)
+  target_compile_options(vosk PRIVATE /bigobj)
+endif()
+
 include(GNUInstallDirs)
 install(TARGETS vosk DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(FILES src/vosk_api.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
