diff --git a/cmake/kaldi-config.cmake.in b/cmake/kaldi-config.cmake.in
index 123f58c56..5bbf521d0 100644
--- a/cmake/kaldi-config.cmake.in
+++ b/cmake/kaldi-config.cmake.in
@@ -1,6 +1,8 @@
 @PACKAGE_INIT@
 
-find_package(Threads)
+include(CMakeFindDependencyMacro)
+find_dependency(unofficial-openfst)
+find_dependency(Threads)
 
 if(NOT TARGET kaldi-base)
     include(${CMAKE_CURRENT_LIST_DIR}/kaldi-targets.cmake)
