diff --git a/CMakeLists.txt b/CMakeLists.txt
index bbf5ad578..f4a72610d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -300,7 +300,7 @@ if(NOT CONDA_ROOT)
     configure_package_config_file(
         ${CMAKE_CURRENT_SOURCE_DIR}/cmake/kaldi-config.cmake.in
         ${CMAKE_CURRENT_BINARY_DIR}/cmake/kaldi-config.cmake
-        INSTALL_DESTINATION lib/cmake/kaldi
+        INSTALL_DESTINATION share/kaldi
     )
     write_basic_package_version_file(
         ${CMAKE_CURRENT_BINARY_DIR}/cmake/kaldi-config-version.cmake
@@ -308,7 +308,7 @@ if(NOT CONDA_ROOT)
         COMPATIBILITY AnyNewerVersion
     )
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/cmake/kaldi-config.cmake ${CMAKE_CURRENT_BINARY_DIR}/cmake/kaldi-config-version.cmake
-        DESTINATION lib/cmake/kaldi
+        DESTINATION share/kaldi
     )
-    install(EXPORT kaldi-targets DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/cmake/kaldi)
+    install(EXPORT kaldi-targets DESTINATION ${CMAKE_INSTALL_PREFIX}/share/kaldi)
 endif()
